### PR TITLE
Improve complexity of assignCase var in CMS Controller

### PR DESCRIPTION
**Pull request by Sinepel**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop 
| Description?      | Improved the complexity of the assignCase variable in the CMS Controller which was set to 1 or 2 without this being made explicit except via a comment.
| Type?             | refacto
| Category?         | FO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | 
| How to test?      | On the front, go to a CMS page and a CMS category and the behaviour should remain unchanged
| Possible impacts? | no

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/24019)

### CODE REVIEW

File: controllers/front/CmsController.php [hunk 1]
```diff
@@ -25,6 +25,9 @@
  */
 class CmsControllerCore extends FrontController
 {
+    const CMS_CASE_PAGE = 1;
+    const CMS_CASE_CATEGORY = 2;
```
Comment (PululuK) [inline controllers/front/CmsController.php]:
```suggestion
    public const CMS_CASE_PAGE = 1;
    public const CMS_CASE_CATEGORY = 2;
```

**Review by PululuK — COMMENTED** [review]
Nice !

**Review by PululuK — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Review by PrestaEdit — APPROVED** [review]

**Comment by boubkerbribri** [comment]
Tests manually by @sarahdib and with UI tests (CMS pages part).
it's QA :heavy_check_mark: .
Thanks @Sinepel

**Comment by PierreRambaud** [comment]
Thank you @Sinepel

**Pull request merged**